tests/device_uvector_tests.cpp

```diff
@@ -199,8 +203,8 @@ TYPED_TEST(TypedUVectorTest, FrontBackElement)
   auto const size{12345};
   rmm::device_uvector<TypeParam> vec(size, this->stream());
 
-  auto const first = TypeParam{42};
-  auto const last  = TypeParam{13};
+  auto const first = TypeParam(42);
```

Comment:
Out of curiosity why did these lines need to change?

Reply:
I added bool to the list of test types to improve coverage. That could go in a follow-up, I was just experimenting and forgot about this.

Reply:
From a PR purity standpoint we should move it to a separate PR. 

But In the end I don't think it matters, so I defer to you. It is not worth holding up this PR

Reply:
I removed all changes that improved code coverage.